Integrations/IronDefense/IronDefense.yml

```diff
@@ -182,15 +310,679 @@ script:
+      description: The destination port associated with this event.
+      type: Number
+    - contextPath: IronDefense.Events.app_domains
+      description: Any domains associated with this event.
+      type: String
+    - contextPath: IronDefense.Events.primary_app_protocol
+      description: The primary application protocol associated with this event.
+      type: String
+    - contextPath: IronDefense.Events.secondary_app_protocol
+      description: The secondary application protocol associated with this event.
+      type: String
+    - contextPath: IronDefense.Events.bytes_in
+      description: The byte count of incoming traffic for this event.
+      type: Number
+    - contextPath: IronDefense.Events.bytes_out
+      description: The byte count of outgoing traffic for this event.
+      type: Number
+    - contextPath: IronDefense.Events.total_bytes
+      description: The byte count of the total traffic (in either direction) for this
+        event.
+      type: Number
+    - contextPath: IronDefense.Events.url
+      description: The related URL for this event, if applicable.
+      type: String
+    - contextPath: IronDefense.Events.raw_data_formats
+      description: The list of distinct raw data formats for this event.
+      type: String
+    - contextPath: IronDefense.Events.src_entity_attribute
+      description: The source entity attribute related to this event, if one could
+        be determined.
+      type: String
+    - contextPath: IronDefense.Events.src_entity_attribute_type
+      description: The source entity attribute type related to this event, if one
+        could be determined.
+      type: String
+    - contextPath: IronDefense.Events.dst_entity_attribute
+      description: The destination entity attribute related to this event, if one
+        could be determined.
+      type: String
+    - contextPath: IronDefense.Events.dst_entity_attribute_type
+      description: The destination entity attribute type related to this event, if
+        one could be determined.
+      type: String
+    - contextPath: IronDefense.Events.vue_url
+      description: The url for displaying the event within IronVUE.
+      type: String
+    - contextPath: IronDefense.Query.GetEvents.constraint.limit
+      description: The maximum number of results that were requested to be returned.
+      type: Number
+    - contextPath: IronDefense.Query.GetEvents.constraint.offset
+      description: The number of results that were skipped - used for paging the results.
+      type: Number
+    - contextPath: IronDefense.Query.GetEvents.constraint.total
+      description: The total number of results possible from the query.
+      type: Number
   dockerimage: demisto/python3:3.7.5.4002
-  isfetch: false
+  feed: false
+  isfetch: true
+  longRunning: false
+  longRunningPort: false
   runonce: false
   script: '-'
-  type: python
   subtype: python3
+  type: python
 tests:
-- IronDefenseTest
+- No tests
```

Comment:
why did you change this

Reply:
I think this just got lost. I'll revert this line